shared/editor/marks/Italic.ts

```diff
@@ -25,29 +25,9 @@ export default class Italic extends Mark {
   }
 
   inputRules({ type }: { type: MarkType }): InputRule[] {
-    /**
-     * Due to use of snake_case strings common in docs the matching conditions
```

Comment:
Tested all these cases and they still function correctly with `markInputRuleForPattern` 💪